kontrol-service/engine/docker.go

```diff
@@ -140,7 +149,60 @@ func processGatewayAndRouteConfigs(gatewayConfigs []apitypes.GatewayConfig, rout
 	return gatewayAndRoutes
 }
 
-func processServiceConfigs(serviceConfigs []apitypes.ServiceConfig, version string) ([]*resolved.Service, []resolved.ServiceDependency, error) {
+func getDeploymentForService(
+	serviceConfig apitypes.ServiceConfig,
+	workloadConfigs []apitypes.DeploymentConfig,
```

Comment:
I think you can replace the `workloadConfigs` var name with `deploymentconfigs`

Reply:
+1